stdlib/public/Darwin/Foundation/Calendar.swift

```diff
@@ -899,15 +899,16 @@ public struct Calendar : Hashable, Equatable, ReferenceConvertible, _MutableBoxi
     
     // MARK: -
     
-    public var hashValue : Int {
-        // We implement hash ourselves, because we need to make sure autoupdating calendars have the same hash
-        if _autoupdating {
-            return 1
-        } else {
-            return _handle.map { $0.hash }
-        }
-    }
-    
+    public func hash(into hasher: inout Hasher) {
+        // We need to make sure autoupdating calendars have the same hash
+	      if _autoupdating {
+            hasher.combine(false)
```

Comment:
Not sure if GitHub or there's inconsistent indentation here, but it would be nice to be consistent.

Reply:
It must be a tabs vs spaces thing; I'll take a look.